packages/hash/shared/src/types.ts

```diff
@@ -1,393 +1,14 @@
-import { BaseUri, VersionedUri } from "@blockprotocol/type-system";
-import slugify from "slugify";
-import { systemUserShortname } from "@hashintel/hash-shared/environment";
-import { frontendUrl } from "./environment";
+// The types exported here are expected to be branded.
+// Perhaps some of these should be put into the subgraph stdlib instead?
```

Comment:
I think some of them already are, (in terms of the upstream subgraph lib we're not publishing/consuming yet).

About branding them there, ... there is a question as to if we should provide branded things in a public API though.  Maybe we just provide branded re-exports here or something.

Reply:
I'd probably delete this line from the code though, no need to commit it.